src/azure-cli/azure/cli/command_modules/apim/_help.py

```diff
@@ -517,3 +517,44 @@
     text: |-
         az apim api versionset delete --resource-group MyResourceGroup --service-name MyServiceName --version-set-id MyVersionSetId
 """
+
+helps['apim api create schema'] = """
+type: command
+short-summary: Create an API Management API Schema.
+parameters:
+  - name: --api-id
+    type: string
+    short-summary: unique name of the api to be created
```

Comment:
It's not the api itself that is going to be created

Reply:
changed description